src/views/Picklist-Detail.vue

```diff
@@ -185,12 +185,16 @@ export default defineComponent({
       const item = this.picklist.pickingItemList.find((product) => product.productId === productId && !product.isChecked)
       if (item) {
         item.isChecked = true;
-        this.lastScannedId = item.id;
+        this.store.dispatch("picklist/updateLastScannedId", item.id)
         // Highlight specific element
         const scannedElement = document.getElementById(item.id);
         scannedElement && (scannedElement.scrollIntoView());
+
+        setTimeout(() => {
+          this.store.dispatch("picklist/updateLastScannedId", "")
+        }, 3000)
       } else {
-        this.lastScannedId = "";
+        this.store.dispatch("picklist/updateLastScannedId", "")
```

Comment:
Clear scanned item it when the picklist is changed and when the user is logged out.